src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1035,6 +1035,13 @@ CHIP_ERROR ConnectivityManagerImpl::_GetEthOverrunCount(uint64_t & overrunCount)
     return GetEthernetStatsCount(EthernetStatsCountType::kEthOverrunCount, overrunCount);
 }
 
+CHIP_ERROR ConnectivityManagerImpl::_ResetEthNetworkDiagnosticsCounts()
+{
+    // On Linux simulation, the packet statistic informations are shared by all running programs,
+    // the current running program does not have permission to reset those counts.
```

Comment:
This could be done by grabbing a starting set of initial counts at start of the connectivity manager impl, and then always reporting difference between current and initial counts. To "reset" counters to zero, you can just replace the initial counts with current counts.

Reply:
Yes, that is a better way to simulate the packet statistic on Linux.